conf/application-json-logger.xml

```diff
@@ -7,7 +7,7 @@
 
```

Comment:
Can you please update the logback-test.xml, logback.xml and remove logger settings from application.conf. Refer [here](https://jira.tools.tax.service.gov.uk/browse/MTDSA-7619) for the changes that are to be made